src/guacd/conf-file.c

```diff
@@ -188,15 +188,21 @@ guacd_config* guacd_conf_load() {
     conf->key_file = NULL;
 #endif
 
+    /* Determine path of configuration file */
+    if(conf_file_path == NULL) {
+        conf_file_path = getenv("GUACD_CONF_FILE");
```

Comment:
Why the change from the constant `GUACD_CONF_FILE` to `getenv("GUACD_CONF_FILE")`?  I believe this changes from using the constant to assuming that it is defined within the environment??

Reply:
This is extension - one can define `"GUACD_CONF_FILE"` environment variable, and if it is not present there, then `GUACD_CONF_FILE` constant is used (as it was done before) - see one line below.

This enables specifying path to config file by "-c" argument or by environment variable (both approaches are quite common, we can discard one of them).

Reply:
Ah, yes, seems I overlooked that part of the original message.